packages/helpers/test/useMountedRef.spec.js

```diff
@@ -0,0 +1,29 @@
+import { renderHook } from "@testing-library/react-hooks";
+
+import useMountedRef from "../src/hooks/useMountedRef";
+
+describe("useMountedRef", () => {
+  test("should increment counter", () => {
```

Comment:
I'm confused by the description "should increment counter". Was it copied from somewhere else?

Reply:
Oh no! I'll fix it